be/src/types/bitmap_value.cpp

```diff
@@ -844,7 +844,9 @@ void BitmapValue::_convert_to_smaller_type() {
             _type = EMPTY;
         } else {
             _type = SINGLE;
-            _sv = _bitmap->minimum();
+            if (auto min_value = _bitmap->minimum(); min_value.has_value()) {
+                _sv = min_value.value();
+            }
```

Comment:
Maybe we should restrict that `min_value` must have value, when `cardinality==1`.
```suggestion
            auto min_value = _bitmap->minimum();
            DCHECK(min_value.has_value());
            _sv = min_value.value();
```